test/e2e/e2e-tests.sh

```diff
@@ -0,0 +1,43 @@
+TEST_NAMESPACE="openshift-marketplace"
```

Comment:
Please add `#!/bin/bash`
Please add `set -e` so that it stops on the first error.